templates/project_add.html

```diff
@@ -29,7 +29,7 @@ <h1 class="text-center">Add Project !</h1>
       <button class="btn bg-dark text-white dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
         Choose tech stack
       </button>
-      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton" style="z-index:100000">

```

Comment:
Why are you adding the style to HTML, instead of using CSS. It's no a big deal, but I believe keeping all styles in the CSS file is a good practice.